src/maestro/Communicator.py

```diff
@@ -695,18 +695,16 @@ def updtr_add(self, name, interval=None, offset=None, push=None, auto=None, perm
                 LDMSD_Req_Attr(attr_id=LDMSD_Req_Attr.INTERVAL, value=str(interval)),
```

Comment:
I want to point out that 'updtr_add' requires the 'interval' attribute. You may want to consider changing the signature of the function. It is orthogonal to the patch, though

Reply:
Thanks for looking at this @nichamon. I'll update the Communicator method to require the interval parameter, and add an example in the config/new_config.yaml template

Reply:
Let's remove the ```if interval``` condition.